src/CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ if ( ICONV_FOUND )
     LIST(APPEND EXTERNAL_INCLUDES ${ICONV_INCLUDE_DIR})
 endif ()
 
+if ( TIRPC_FOUND )
+    LIST(APPEND EXTERNAL_LIBRARIES ${TIRPC_LIBRARIES})
```

Comment:
This doesn't actually match what gets produced by FindTIRPC.cmake (LIBRARIES vs LIBRARY) - the build fails to link.